src/WCSimWCTrigger.cc

```diff
@@ -595,3 +609,29 @@ void WCSimWCTriggerNDigits2::DoTheWork(WCSimWCDigitsCollection* WCDCPMT) {
   bool ndigits_test = true;
   AlgNDigits(WCDCPMTCopy, remove_hits, ndigits_test);
 }
+
+// *******************************************
+// DERIVED CLASS
+// *******************************************
+
+
+WCSimWCTriggerNoTrigger::WCSimWCTriggerNoTrigger(G4String name,
```

Comment:
I think this is from another pull request which has already been merged (#220). Or is there some change here that goes with this PR? If not, please update so this change is removed.

Reply:
Yes I had to add inside the mother class the detector element to instantiate separate trigger algorithm, I have also cleaned the file.